libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -1118,6 +1149,8 @@ IotMqttError_t IotMqtt_Connect( const IotMqttNetworkInfo_t * pNetworkInfo,
         /* Set the MQTT packet serializer overrides. */
         #if IOT_MQTT_ENABLE_SERIALIZER_OVERRIDES == 1
             pNewMqttConnection->pSerializer = pNetworkInfo->pMqttSerializer;
+        #else
+            pNewMqttConnection->pSerializer = NULL;
```

Comment:
Don't have to do this, the MQTT connection is memset to 0 upon creation.

Reply:
We will do it anyway to make sure the intent of the code is clear to the reader.